bundles/org.openhab.binding.fox/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fox"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	
+	<thing-type id="system">
+		<label>Fox</label>
+		<description>Nexwell Fox System</description>
+		<channels>
+			<channel id="taskCommand" typeId="taskRequest" />
+			<channel id="taskState" typeId="taskExecuted" />
+			<channel id="resultTrigger" typeId="resultNotice" />
+			<channel id="resultState" typeId="resultAcquired" />
+		</channels>
+		<config-description>
+			<parameter name="gateHost" type="text" required="true">
+				<label>Connection address</label>
+				<description>Fox System net address - IP or domain, with an optional port (default: 23).</description>

```

Comment:
Better use a dedicated config parameter for the port and apply the default value of 23 to it. That also simplifys code.